audit-test-tools/template.yaml

```diff
@@ -0,0 +1,126 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: Audit accoun test tool infrastructure
+Transform: AWS::Serverless-2016-10-31
+
+Parameters:
+  CodeSigningConfigArn:
+    Description: The ARN of the Code Signing Config to use, provided by the deployment pipeline
+    Type: String
+    Default: none
+  Environment:
+    Description: The environment type
+    Type: String
+    AllowedValues:
+      - dev
+      - build
+      - staging
+  PermissionsBoundary:
+    Description: The ARN of the permissions boundary to apply to any role created by the template
+    Type: String
+    Default: none
+
+Conditions:
+  UseCodeSigning: !Not [!Equals [!Ref CodeSigningConfigArn, none]]
+  UsePermissionsBoundary: !Not [!Equals [!Ref PermissionsBoundary, none]]
+
+Globals:
+  Function:
+    CodeSigningConfigArn:
+      !If [UseCodeSigning, !Ref CodeSigningConfigArn, !Ref AWS::NoValue]
+    CodeUri: dist/
+    Environment:
+      Variables:
+        NODE_OPTIONS: --enable-source-maps
+    PermissionsBoundary:
+      !If [UsePermissionsBoundary, !Ref PermissionsBoundary, !Ref AWS::NoValue]
+    ReservedConcurrentExecutions: 10
+    Runtime: nodejs16.x
+    Timeout: 30
+
+Resources:
+  WriteTestDataToAthenaBucketFunction:
+    #checkov:skip=CKV_AWS_115:Defined in Globals section
+    #checkov:skip=CKV_AWS_116:The SQS queue itself will be responsible for its own DLQ
+    #checkov:skip=CKV_AWS_117:VPC not required
+    Type: AWS::Serverless::Function
+    Properties:
+      Handler: writeTestDataToAthenaBucket.handler
+      Role: !GetAtt WriteTestDataToAthenaBucketFunctionRole.Arn
+      Environment:
+        Variables:
+          ATHENA_OUTPUT_BUCKET_NAME: '{{resolve:ssm:AthenaQueryOutputBucketName}}'
+          QUERY_COMPLETED_QUEUE_URL: '{{resolve:ssm:QueryCompletedQueueUrl}}'
+      FunctionName: !Sub ${AWS::StackName}-write-test-data-to-athena-bucket
+      KmsKeyArn: '{{resolve:ssm:LambdaKmsKeyArn}}'
+      Events:
+        QRIntegrationTestsTrigggerEvent:
+          Type: SQS
+          Properties:
+            Queue: !Sub 'arn:aws:sqs:${AWS::Region}:{{resolve:ssm:QueryResultsAccountNumber}}:txma-query-results-${Environment}-write-test-data-to-athena-bucket-queue'
```

Comment:
Is the queue name correct? I see `txma-qr-infra-build-write-test-data-to-athena-bucket-queue` in the query results build account for example.

Reply:
Ah no, you're right, it wasn't! I've corrected it now.